google/ads/google_ads/config.py

```diff
@@ -129,6 +129,25 @@ def load_from_yaml_file(path=None):
 
     return parse_yaml_document_to_dict(yaml_doc)
 
+@_config_validation_decorator
+@_config_parser_decorator
+def load_from_dict(config_dict):
+    """ Check if the argument is dictionary or not
```

Comment:
Add a little more detail here. This function does more than just check the type of the input, it also returns it to the caller in a way that invokes the decorators that parse it's values and validates the presence of keys.